blogs/2023/07/20/mangling-vscode.md

```diff
@@ -0,0 +1,174 @@
+---
+Order: 83
+TOCTitle: How we cut 20% off the size of VS Code's JavaScript
+PageTitle: How we cut 20% off the size of VS Code's JavaScript
+MetaDescription: A look at how we how we cut 20% off the size of VS Code's JavaScript with name mangling.
+Date: 2023-07-20
+Author: Matt Bierner
+---
+# How we cut 20% off the size of VS Code's JavaScript
```

Comment:
This H1 is also long and will likely wrap. Good to keep the H1 below 40 characters.
Maybe something like "Shrinking VS Code with JS name mangling"